### PR TITLE
[RHDEVDOCS - 3045]: Additional tutorial updates for Pipelines 1.5

### PR DESCRIPTION
**Pull request by sounix000**
- **Aligned team**: Dev Tools
- **OCP version for cherry-picking**: `enterprise-4.8`, `enterprise-4.9`
- **JIRA issues**: [Tutorial updates 1.5](https://issues.redhat.com/browse/RHDEVDOCS-3045)
- **Preview pages**: [Running a pipeline](https://deploy-preview-34897--osdocs.netlify.app/openshift-enterprise/latest/cicd/pipelines/creating-applications-with-cicd-pipelines?utm_source=github&utm_campaign=bot_dp#running-a-pipeline_creating-applications-with-cicd-pipelines)
- **SME+QE review**: Praveen Thangadacha, Veeresh Acharya
- **Peer-review**: Preeti Chandrashekar
- **Additional info**: 
  - This PR was raised in response to https://github.com/openshift/pipelines-tutorial/pull/145.
  - The main PR for Pipelines 1.5 tutorial is https://github.com/openshift/openshift-docs/pull/34675.

### CODE REVIEW

File: modules/op-running-a-pipeline.adoc [hunk 1]
```diff
@@ -19,6 +19,7 @@ $ tkn pipeline start build-and-deploy \
     -p deployment-name=pipelines-vote-api \
     -p git-url=https://github.com/openshift/pipelines-vote-api.git \
     -p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/pipelines-vote-api
```
Comment (VeereshAradhya) [inline modules/op-running-a-pipeline.adoc:21]: `-p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/pipelines-vote-api \`, `\` is missing

File: modules/op-running-a-pipeline.adoc [hunk 2]
```diff
@@ -41,6 +42,7 @@ $ tkn pipeline start build-and-deploy \
     -p deployment-name=pipelines-vote-ui \
     -p git-url=https://github.com/openshift/pipelines-vote-ui.git \
     -p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/pipelines-vote-ui
```
Comment (VeereshAradhya) [inline modules/op-running-a-pipeline.adoc:44]: `-p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/pipelines-vote-api \`, `\` is missing

**Review by VeereshAradhya — APPROVED** [review]

**Review by vdemeester — APPROVED** [review]
/lgtm

**Pull request merged**